pkg/util/nsenter/nsenter.go

```diff
@@ -119,6 +119,6 @@ func (ne *Nsenter) AbsHostPath(command string) string {
 
 // SupportsSystemd checks whether command systemd-run exists
 func (ne *Nsenter) SupportsSystemd() (string, bool) {
-	systemdRunPath, hasSystemd := ne.paths["systemd-run"]
-	return systemdRunPath, hasSystemd
+	systemdRunPath := ne.paths["systemd-run"]
+	return systemdRunPath, systemdRunPath != ""
```

Comment:
Just recheck this.

The default `systemd-run` binary file will be `/systemd-run`, if not found in any of `/bin`, `/usr/sbin`, and `/usr/bin`.

So `ne.paths["systemd-run"]` will not be empty string.